### PR TITLE
Blip2 model for image-to-text

### PR DESCRIPTION
**Pull request by sharma-riti**
Pipeline example: [pipeline](https://ml.azure.com/experiments/id/12a39b97-7af2-4b5e-9b8c-c24431299802/runs/shy_station_6z010740f4?wsid=/subscriptions/381b38e9-9840-4719-a5a0-61d9585e1e91/resourceGroups/automlimage_eastus2_rg/providers/Microsoft.MachineLearningServices/workspaces/risha-ws-eastus2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)
Model in registry - [model](https://ml.azure.com/registries/automl-images-dev/models/salesforce-blip2-opt-2-7b/version/2?tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview)

### CODE REVIEW

File: assets/models/system/salesforce-blip2-opt-2.7b/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: salesforce-blip2-opt-2.7b
+path: ./
+properties:
+    SHA: 6e723d92ee91ebcee4ba74d7017632f11ff4217b
+    inference-min-sku-spec: 2|0|7|14
+    inference-recommended-sku: Standard_DS2_v2, Standard_D2a_v4, Standard_D2as_v4, Standard_DS3_v2,
```
Comment (raduk) [inline assets/models/system/salesforce-blip2-opt-2.7b/spec.yaml:7]: what is the latency on CPU skus?

File: assets/models/system/salesforce-blip2-opt-2.7b/description.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+         ["image1"],
+         ["image2"]
+      ]
+   }
+}
+```
+Note:
+- "image1" and "image2" should be publicly accessible urls or strings in `base64` format.
+
+#### Sample output
```
Comment (raduk) [inline assets/models/system/salesforce-blip2-opt-2.7b/description.md:33]: can we add an image similar to the other models so we make the model card more appealing?

https://ml.azure.com/models/microsoft-swinv2-base-patch4-window12-192-22k/version/9/catalog/registry/azureml?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourceGroups/rbhimani-rg/providers/Microsoft.MachineLearningServices/workspaces/rbhimani-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

**Review by PhaniShekhar — DISMISSED** [review]

File: assets/models/system/salesforce-blip2-opt-2.7b/description.md (hunk 2 above)
Comment (sharma-riti) [reply assets/models/system/salesforce-blip2-opt-2.7b/description.md:33]: added

**Review by haritha-bendapudi — APPROVED** [review]

**Review by PhaniShekhar — DISMISSED** [review]
If you want to use an image depicting a real-life scenario for illustration purposes, you can use images from here: https://brandcentral.microsoft.com/

**Review by raduk — APPROVED** [review]

**Review by haritha-bendapudi — APPROVED** [review]

**Review by PhaniShekhar — APPROVED** [review]

**Pull request merged**